.gitignore

```diff
@@ -1,23 +1,10 @@
-
-# Binaries for programs and plugins
-*.exe
-*.exe~
-*.dll
-*.so
-*.dylib
-bin/scylla-operator
-dist/
```

Comment:
This is folder used by the goreleaser, let's leave it here until we get rid of it

Reply:
got rid of goreleaser as we planned